libevm/tooling/release/README.md

```diff
@@ -0,0 +1,26 @@
+# Release
+
+In the following, we create a release candidate tag `v1.13.14-0.2.0.rc.4`.
+
+1. Create two branches, usually from the `main` branch:
+
+    ```bash
+    git fetch origin main:main
+    git checkout main
+    git checkout -b release/v1.13.14-0.2.0.rc.4
```

Comment:
Lets add a step before that sets an environment variable and then use it in the rest. That helps someone copy-paste with fewer modifications.